File: HomeFinance.Web/ClientApp/src/app/Models/closing-pay-list-view.model.ts
export class ClosingPayListData {	
    id: number;
    payDate: Date;
	storeName: string;
	amount: number;
    note: string;
    closingDate: Date;
}

export class ClosingPayListView {	
    rowCount: number;
    closingPayListData: ClosingPayListData[];
}

File: HomeFinance.Application/StorePays/Queries/ClosingPayListViewModel.cs
using System;
using System.Collections.Generic;

namespace HomeFinance.Application.StorePays.Queries
{
    public class ClosingPayListViewModel
    {
        public int RowCount { get; set; }
        public List<ViewClosingPayModel> ClosingPayListData { get; set; }
    }

    public class ViewClosingPayModel
    {
        public int Id { get; set; }
        public DateTime PayDate { get; set; }
        public string StoreName { get; set; }
        public decimal Amount { get; set; }
        public string Note { get; set; }
    }
}
